src/Driver.ts

```diff
@@ -42,8 +42,7 @@ export class Driver {
       this.get(LOCKER_TEST_KEY)
       this.remove(LOCKER_TEST_KEY)
     } catch (e) {
-      if (__DEV__)
-        console.error(e)
+      __DEV__ && console.error(e)
```

Comment:
`&&` shouldn't be used in this case as it's better suited for boolean checks not control flow. Prefer if/else for control flow